cmd/enval.go

```diff
@@ -96,12 +96,24 @@ func executeCmd(_ *cobra.Command, _ []string) error {
 		os.Exit(1)
 	}
 
-	_, err = toolsManager.ValidateManifestAndNotify(*manifest, cmdNotifier)
+	results, err := toolsManager.ValidateManifestAndNotify(*manifest, cmdNotifier)
 	if err != nil {
 		prettyPrintError(err)
 		os.Exit(1)
 	}
-
+	ok := true
+	for _, result := range results {
+		errs := result.CheckErrors()
+		for _, err := range errs {
+			{
+				ok = false
+				prettyPrintError(err)
```

Comment:
the info was already printed. This must just exit 1 without printing as soon as one is not valid